extensions/amp-analytics/0.1/linker-manager.js

```diff
@@ -117,6 +120,15 @@ export class LinkerManager {
         });
   }
 
+  /**
+   * Remove any listeners created to manage form submission.
+   */
+  removeListeners() {
```

Comment:
let's call it `dispose()` to pair up with `init()`

Reply:
Done.